src/Parser.js

```diff
@@ -66,11 +66,17 @@ export function parse (value, points, declarations, rules, rulesets) {
 					default:
 						push(rulesets, ruleset(temporary, points, index, offset, rules, type, props = [], children = []))
 
-						if (character !== 59)
-							if (offset === 0)
+						if (character === 59) break
```

Comment:
This is still incomplete, it breaks on other obscure `@at-rules`. I came up with the following

```js
if (character !== 59)
	if (offset === 0)
		parse(temporary, points, children, props, rulesets)
	else if (previous !== 109 && previous !== 115)
		parse(temporary, points, children, [''], children)
	else if (parse(value, points, props = [], rules, children), sizeof(props))
		push(children, ruleset(value, points, 0, 0, rules, type, rules, props))
```

with an amendment to this line(L96): https://github.com/thysultan/stylis.js/blob/8451caba2a67a65ad56a2e1c713a3b8e592ecfa5/src/Parser.js#L96

into the following

```js
character = peek(0), offset = strlen(type = temporary += identifier(caret()))
```